### DISCUSSION TITLE
Hide the portal link

### DISCUSSION THREAD

**Discussion by casualmatt** [Budibase/budibase#3903] [Help]
Hello, 
do you how i could hide the link to the portal?
![image](https://user-images.githubusercontent.com/85650530/147832611-a06684e8-78a0-4b4b-9898-65183947e4d5.png)

I use budibase only for the front end, and i don't want the user to see the login page of Budibase. 

Ty have a happy new year

**Comment by shogunpurple** [comment]
Hi @casualmatt - you cannot do this with the standard navigation layout, as the idea is that users who are using the navigation layout can go back to the portal to view all of the applications they have access to. If they aren't a builder or admin, they won't see the portal that you see, only a list of their budibase apps.

You can however use the empty layout to do this, but then you will have to add all the elements that you get with the navigation layout yourself.

https://user-images.githubusercontent.com/11256663/147832831-cb73ac17-58f0-4cac-ab06-1939c0f313d9.mov


Happy new year!
